lib/cartopy/crs.py

```diff
@@ -1134,6 +1141,35 @@ def __init__(self, central_longitude=-96.0, central_latitude=39.0,
         self._x_limits = bounds[0], bounds[2]
         self._y_limits = bounds[1], bounds[3]
 
+    @classmethod
+    def from_proj4(cls, proj4_dict, **kwargs):
```

Comment:
I'd want to assert that the ``proj`` parameter is set correctly (in this case, ```lcc```).